Client/mods/deathmatch/logic/CClientEntity.cpp

```diff
@@ -175,19 +175,10 @@ CClientEntity::~CClientEntity ( void )
     g_pCore->UpdateDummyProgress();
 
     // Remove from m_DisabledCollisions
-    switch ( this->GetType () )
```

Comment:
Does someone understand why this (without "this->" too) gives a "LNK2019" error?
Maybe because it's CClientVehicle is getting destructed before, so GetType is not implemented for the element anymore?

Reply:
Calling any virtual member function in a destructor is a very bad idea.

Reply:
GetType is virtual

Reply:
So either not check the type and do it for all element-types or get the type in another way.